device/chip/tlb_manager.h

```diff
@@ -0,0 +1,46 @@
+/*
+ * SPDX-FileCopyrightText: (c) 2024 Tenstorrent Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
Don't have to do it now, but it might be worth adding a bit of documentation about what is TLB, how they are used, why we have them etc.

Reply:
We have some internal docs that we could convert to md files. I think it would be nice to do it in parallel with designing this TLB manager class. Not saying that Bojan has to do it, I have played around with TLBs as well earlier...https://github.com/tenstorrent/tt-umd/issues/429

Reply:
Captured in an issue. @pjanevskiTT please fill out the mentioned references in the issue.